CMakeLists.txt

```diff
@@ -240,6 +242,9 @@ if(CLR_CMAKE_PLATFORM_XPLAT)
         # -Wno-return-type # switch unreachable code
         # -Wno-switch # switch values not handled
 
+    add_compile_options(-fexceptions)
+    add_compile_options(-fstack-protector-all)
```

Comment:
Do we really want all? or should -fstack-protector sufficient?
What's the perf impact with -fstack-protector-all?